content/documentation/pages/6-recipes/7-scaling/1-manual-scaling.md

```diff
@@ -0,0 +1,118 @@
+
+![SCDF Manual Scaling](images/scdf-scale-manual.png)
+
+This manual approach allows the Operators to intervene and scale out when they observe high load. It also can be useful to scale the pipelines in case of an expected high load events. For example a retail store would pre-schedule scale out for events such as Black Friday and alike.
+
+When a more sophisticated approach is required to control the scale-out and scale-in operations in an automated manner, there's an option to leverage SCDF monitoring support with Prometheus. Check the [Autoscale streaming data pipelines with SCDF and Prometheus](%currentPath%/recipes/scaling/autoscaling/) recipe to learn how to do this.
+
+## Prerequisite
+
+This recipe uses the Kubernetes platform. Follow the [Kubectl](%currentPath%/installation/kubernetes/kubectl/) or the [Helm](%currentPath%/installation/kubernetes/helm/) installation instructions to setup Spring Cloud Data Flow with Kafka broker.
+
+```shell
+helm install --name my-release stable/spring-cloud-data-flow --set kafka.enabled=true,rabbitmq.enabled=false,kafka.persistence.size=10Gi
+```
+
+And register the latest [kafka-docker](https://dataflow.spring.io/kafka-docker-latest) app starters.
+
+Start a SCDF Shell and connect it to your Data Flow Server:
+
+```shell
+server-unknown:>dataflow config server http://<SCDF IP>
+```
+
+## Scaling Recipe
+
+### Create stream pipeline
+
+```shell
+stream create --name scaletest --definition "time --fixed-delay=995 --time-unit=MILLISECONDS | transform --expression=\"payload + '-' + T(java.lang.Math).exp(700)\" | log"
+```
+
+The `time` source generates current timestamp messages on a fixed time-interval (995ms = ~1msg/s), the `transform` processor performs a math operation to simulate a high CPU processing, and the `log` sink prints the transformed message payload.
+
+### Deploy pipeline with data partitioning
+
+```shell
+stream deploy --name scaletest --properties "app.time.producer.partitionKeyExpression=payload,app.transform.spring.cloud.stream.kafka.binder.autoAddPartitions=true,app.transform.spring.cloud.stream.kafka.binder.minPartitionCount=4"
+```
+
+The `producer.partitionKeyExpression=payload` property configures time’s output binding for partitioning. The partition key expression uses the message payload (e.g. the toString() value of the current timestamp) to compute how the data needs to be partitioned to the downstream output channels.
+The `spring.cloud.stream.kafka.binder.autoAddPartitions` deployment property instructs the Kafka binder to create new partitions when required. This is required if the topic is not already over-partitioned.
+The `spring.cloud.stream.kafka.binder.minPartitionCount` property sets the minimum number of partitions that the Kafka binder configures on the topic, which is where the transform-processor is subscribing for new data.
+
+<!--TIP-->
+
+When using RabbitMQ, the `app.log.spring.cloud.stream.kafka.binder.*` deployment properties are not needed.
```

Comment:
As suggested previously, let's remove any rabbit reference. This content is strictly only for SCDF + K8s + Kafka anyway.